src/lib/eina/eina_sched_posix.c

```diff
@@ -0,0 +1,70 @@
+/* EINA - EFL data type library
+ * Copyright (C) 2010 ProFUSION embedded systems
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library;
+ * if not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "eina_sched.h"
+#include <sched.h>
+#include <sys/time.h>
+#include <sys/resource.h>
+#include <errno.h>
+#include <pthread.h>
+
+inline void _eina_sched_prio_drop(void)
```

Comment:
should do i keep the _? because eina_sched.c ->eina_sched_prio_drop() call _eina_sched_prio_drop();

Reply:
Why `eina_sched_prio_drop` calls `_eina_shced_prio_drop`? Just let  `eina_sched_{win32/posix}` implement `eina_sched_prio_drop` instead of `_eina_sched_pro_drop`

Reply:
I was maintaining what already existed